solr/solr-ref-guide/src/requesthandlers-and-searchcomponents-in-solrconfig.adoc

```diff
@@ -169,3 +169,14 @@ Many of the other useful components are described in sections of this Guide for
 * `TermVectorComponent`, described in the section <<the-term-vector-component.adoc#the-term-vector-component,The Term Vector Component>>.
 * `QueryElevationComponent`, described in the section <<the-query-elevation-component.adoc#the-query-elevation-component,The Query Elevation Component>>.
 * `TermsComponent`, described in the section <<the-terms-component.adoc#the-terms-component,The Terms Component>>.
+* `RealTimeGetComponent`, described in the section <<realtime-get.adoc#realtime-get,RealTime Get>>.
+* `ClusteringComponent`, described in the section <<result-clustering.adoc#result-clustering,Result Clustering>>.
+* `SuggestComponent`, described in the section <<suggester.adoc#suggester,Suggester>>.
+* `AnalyticsComponent`, described in the section <<analytics.adoc#analytics,Analytics>>.
+
+Other components that ship with Solr include:
+
+* `ResponseLogComponent`, used to record which documents are returned to the user via the Solr log, described in the {solr-javadocs}solr-core/org/apache/solr/handler/component/ResponseLogComponent.html[ResponseLogComponent] javadocs.
+* `PhrasesIdentificationComponent`, used to identify & score "phrases" found in the input string, based on shingles in indexed fields, described in the {solr-javadocs}solr-core/org/apache/solr/handler/component/PhrasesIdentificationComponent.html[PhrasesIdentificationComponent] javadocs.
+
+Lastly, you may be interested in some other components created by the community and listed on the https://solr.cool/#searchcomponents[Solr Cool] website.
```

Comment:
It's great to mention outside resources, though I'd like to point out that SearchComponent is merely one of *many* abstractions you could write a plugin for.  Maybe we should mention "Solr Cool" in exactly one suitable place (about Solr plugins generally) instead of potentially many places.

Reply:
I was wondering about actually linking to the specific types of plugins on the Solr Cool site? 

https://github.com/solr-extensions/solr-extensions.github.io/issues/19

I was wondering about not adding that link in this PR, and doing a separate one with links from the relevant portions of the ref guide.   Or maybe we need to have one place in ref guide with "related projects" or something?   I'm hopeful that solr.cool ends up being a clearing house of community created extensions to Solr of all kinds ;-)

Reply:
Yeah let's separate external plugin advocacy to another issue and address that comprehensively.

Reply:
Makes sense.